docs/accelerometer.rst

```diff
@@ -8,31 +8,36 @@ also provides convenience functions for detecting gestures. The
 recognised gestures are: ``up``, ``down``, ``left``, ``right``, ``face up``,
 ``face down``, ``freefall``, ``3g``, ``6g``, ``8g``, ``shake``.
 
+By default MicroPython sets the accelerometer to +/- 2g, and this can only be
+modified in the DAL layer below. The accelerometer returns a value in the range 
+0..1024 for each axis, which is then scaled accordingly.
 
 Functions
 =========
 
 .. py:function:: get_x()
 
     Get the acceleration measurement in the ``x`` axis, as a positive or
-    negative integer, depending on the direction.
-
+    negative integer, depending on the direction. The measurement is given in 
+    milli-g, for example +/- 2g will return +/- 2000mg. 
```

Comment:
`for example +/- 2g will return +/- 2000mg.`
I know it's been explained at the top of the page, but some people will probably miss it, and the accelerometer range is not really something that has been properly explained in general (it'll probably be new to a lot of people).
Could we rephrase this one (and the others below) to `by default the accelerometer is configured with a range of +/- 2g, and so this method will return a value between +/- 2000mg`?